src/main/test/mate/academy/dao/impl/RoleDaoImplTest.java

```diff
@@ -0,0 +1,52 @@
+package mate.academy.dao.impl;
+
+import java.util.NoSuchElementException;
+
+import mate.academy.dao.RoleDao;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.model.Role;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RoleDaoImplTest extends AbstractDaoTest {
+    private RoleDao roleDao;
+    private Role userRole;
+    private Role expectedRole;
```

Comment:
you use this variable in one test method, make it local. Look through other classes and check if you can to do the same for other fields